packages/scaffold-ui/src/partials/w3m-account-default-widget/styles.ts

```diff
@@ -17,6 +17,11 @@ export default css`
     margin-bottom: var(--wui-spacing-3xs);
   }
 
+  wui-tag {
+    position: absolute;
```

Comment:
We should avoid absolute positioning, this might result unexpected stylings in different devices / different screen sizes - yet we don't control the vertical positioning here.

As with the other `wui-list-items`, the text should have `flex:1`, this will let the `wui-tag` to be on the right.

You can achieve this by giving the `wui-list-item > wui-text { flex: 1 }`. But first please check why `wui-text` doesn't have `flex:1` since in it's style it should already have that property.